src/styles/display.style.js

```diff
@@ -0,0 +1,29 @@
+import {StyleSheet} from 'react-native';
+
+const OVERFLOW_MARGIN = 50;
+
+const overflowVisibleTrick = {
```

Comment:
By the name of the variable, I guess there is some dark magic involved in this code, can you put a comment about it?

Reply:
I just saw the comment in this file: 

https://github.com/BearStudio/start-ui-native/pull/10/files#diff-3baeb577ca098741f2b8a0e908574f234f4f379841ff59f0ca7117a38320fbcaR20

A comment here would be useful I think.

Reply:
I agree with you, I added the comments